README.md

```diff
@@ -4,7 +4,7 @@
 
 ----
 
-[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides elegant, concise construction of versatile graphics, and affords high-performance interactivity over large or streaming datasets. Bokeh can help anyone who would like to quickly and easily make interactive plots, dashboards, and data applications.
+[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides elegant, concise construction of versatile graphics and affords high-performance interactivity across large or streaming datasets. Bokeh can help anyone who wants to create interactive plots, dashboards, and data applications quickly and easily.
```

Comment:
BTW @VroomRk we have tried to keep this "elevator pitch" consistent across all the bokeh.org properties.  After this PR is merged, if you could submit a PR to update the "front page" to match, that would be appreciated:

https://github.com/bokeh/bokeh.org/blob/2f082bb9fa8691547c593d5e6790b74356b7aa97/_includes/roadmap.html#L12-L14

That would be appreciated. There may also be a similar statement to update in the Sphinx source for the docs page too.

Reply:
Sure @bryevdv I will submit another PR to make these changes 👍